### ISSUE TITLE
Accordion - JS warning

### ISSUE THREAD

**Issue by AndrePinto-NET** [DevExpress/devextreme-react#129]
I'm using the latest version of devextreme (18.1.4-pre-18155), with the latest version of devextreme-react (18.1.4-alpha.7), and with the accordion widget, I get a warning on the JS file generated from my TSX file:

> Line 19:  Unexpected mix of '&&' and '||'  no-mixed-operators

> Line 29:  Unexpected mix of '&&' and '||'  no-mixed-operators

The code it self is the following:

```
function AccordionTitleTemplate() {
        return _super !== null && _super.apply(this, arguments) || this;
    }
```

```
function AccordionContentTemplate() {
        return _super !== null && _super.apply(this, arguments) || this;
    }
```
Everything works, at least I saw no issues, I just would like to get ride of the warning.

**Comment by AndrePinto-NET** [comment]
Why not just add parenthesis to the first && and permanently solve that warning?

`return (_super !== null && _super.apply(this, arguments)) || this;`

**Comment by lukyanovas** [comment]
@AndrePinto-NET
As far I understand these warnings occur in your TS-generated code, isn't it?

**Comment by lukyanovas** [comment]
Yes, I see. This code is automatically generated by the TS compiler, and I see no need to fix anything in it. By the way, most projects omit linting the `node_modules` folder. Do you have any strong reasons for doing so?

**Comment by AndrePinto-NET** [comment]
Some additional info here. I did not specified a constructor on my class, so by default, TS writes on the generated java script file the following constructor:

`    function MyClass() {
        return _super !== null && _super.apply(this, arguments) || this;
    }`

If I write the constructor my self, it ends up:

`function MyClass(props) {
        return _super.call(this, props) || this;
    }`

This will remove the warning that I was talking about. I can clearly see, like you mentioned before, that this has nothing to do with devextreme it self.
